### PR TITLE
misc: Cluster, project, environment read and beans

### PR DESCRIPTION
**Pull request by ayu-devtron**
# Description

Fixes #

## Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit/api test cases.

## Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/environment/EnvironmentService.go [hunk 1]
```diff
@@ -14,14 +14,17 @@
  * limitations under the License.
  */
 
-package cluster
+package environment
```
Comment (kartik-579) [inline pkg/environment/EnvironmentService.go:17]: i think we should keep this at /pkg/cluster/environment instead of /pkg/cluster

**Review by kartik-579 — CHANGES_REQUESTED** [review]

**Comment by kartik-579** [comment]
checked structure wise but did not check line by line code assuming all code was shifted and new is not written. please let me know if anything else needs to be checked

**Comment by ayu-devtron** [comment]
> checked structure wise but did not check line by line code assuming all code was shifted and new is not written. please let me know if anything else needs to be checked

yup, no new code is written

File: pkg/environment/EnvironmentService.go (hunk 1 above)
Comment (ayu-devtron) [reply pkg/environment/EnvironmentService.go]: done

**Review by vikramdevtron — APPROVED** [review]

**Pull request merged**